language/bytecode_verifier/src/type_memory_safety.rs

```diff
@@ -807,8 +888,7 @@ impl<'a> TypeAndMemorySafetyAnalysis<'a> {
                 }
             }
 
-            // TODO: Handle type actuals for generics
-            Bytecode::MoveFrom(idx, _) => {
+            Bytecode::MoveFrom(idx, tys_idx) => {
```

Comment:
I thought we were not allowing instances of generic resources to be published.  At least, that is what I remember from a conversation with @tnowacki.  Could it be that we are allowing only types that are nominally declared to be resource published?  Perhaps because for these types we are guaranteed that they would be resources.

Reply:
I think the latest consensus is that we will allow generic resources to be published. However right now we're not supporting it fully (see #723).

Yes we are only going to allow nominal resources to be published.